test/remote/gateways/remote_plexo_test.rb

```diff
@@ -31,6 +31,22 @@ def setup
       description: 'Test desc',
       reason: 'requested by client'
     }
+
+    @network_token_credit_card = ActiveMerchant::Billing::NetworkTokenizationCreditCard.new({
+        first_name: 'Santiago', last_name: 'Navatta',
```

Comment:
💬 Style:

Silly thing => `first_name` and `last_name` are in the same line but the ramaining attributes are one per line